tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -116,12 +116,6 @@ static cl::opt<VersionNumber> MaxSPIRVVersion(
                clEnumValN(VersionNumber::SPIRV_1_6, "1.6", "SPIR-V 1.6")),
     cl::init(VersionNumber::MaximumVersion));
 
-static cl::list<std::string>
```

Comment:
I am not sure if we should be removing this option. Anything that will modify the current behavior of the translator must be avoided. May be I am missing something here?

Thanks